packages/core/src/modules/proofs/ProofsModule.ts

```diff
@@ -1,32 +1,30 @@
 import type { DependencyManager, Module } from '../../plugins'
-import type { ProofsModuleConfigOptions } from './ProofsModuleConfig'
+
+import { RevocationNotificationService } from '../credentials/protocol/revocation-notification/services'
 
 import { ProofsApi } from './ProofsApi'
-import { ProofsModuleConfig } from './ProofsModuleConfig'
+import { IndyProofFormatService } from './formats/indy/IndyProofFormatService'
+import { V1ProofService } from './protocol/v1'
+import { V2ProofService } from './protocol/v2'
 import { ProofRepository } from './repository'
-import { ProofService } from './services'
 
 export class ProofsModule implements Module {
-  public readonly config: ProofsModuleConfig
-
-  public constructor(config?: ProofsModuleConfigOptions) {
```

Comment:
We should keep the config

Reply:
I assume it needs the registerInstance call